ted_sws/notice_transformer/services/notice_transformer.py

```diff
@@ -0,0 +1,56 @@
+import abc
+import pathlib
+import tempfile
+
+from ted_sws.data_manager.adapters.mapping_suite_repository import MappingSuiteRepositoryInFileSystem
+from ted_sws.domain.model.manifestation import RDFManifestation
+from ted_sws.domain.model.notice import Notice
+from ted_sws.domain.model.transform import MappingSuite
+from ted_sws.notice_transformer.adapters.rml_mapper import RMLMapperABC
+
+
+class NoticeTransformerABC(abc.ABC):
+    """
+        This class is a general interface for transforming a notice.
+    """
+    @abc.abstractmethod
+    def transform_notice(self, notice: Notice):
+        """
+            This method performs the transformation on a notice.
+        :param notice:
+        :return:
+        """
+
+
+class NoticeTransformer(NoticeTransformerABC):
+    """
+        This class is a concrete implementation of transforming a notice using rml-mapper.
+    """
+
+    def __init__(self, mapping_suite: MappingSuite, rml_mapper: RMLMapperABC):
+        self.mapping_suite = mapping_suite
+        self.rml_mapper = rml_mapper
+
+
+    def _inject_in_package_notice_xml_manifestation(self, package_path: pathlib.Path, notice: Notice):
+        mapping_suite_repository = MappingSuiteRepositoryInFileSystem(repository_path=package_path.parent)
+        mapping_suite_repository.add(mapping_suite=self.mapping_suite)
+        notice_path = package_path / "transform" / "resources" / "notice.xml"
```

Comment:
OUCH! This should be very very well documented. 

I believe, having a dedicated `input` (next to `test_data`) folder would be much cleaner and more explicit for everyone.